docs/tutorial_tests.md

```diff
@@ -13,7 +13,8 @@ Let's say we want to create a Test where we send a bunch of questions to the SUT
 To run these questions as a Test in NewHELM, we need (for now) to create a class for our Test. Let's call our Test `DemoSimpleQATest`.
 Since this fits as a [Prompt Response Test](prompt_response_tests.md) we can have it inherit from `BasePromptResponseTest`. We now have several abstract methods we need to define.
 
-The first phase in a PromptResponseTest is making the `TestItem`s. We want these to be our questions from `an_example.jsonl`. NewHELM uses [DependencyHelper](../newhelm/dependency_helper.py) to ensure good hygiene of data dependencies (e.g. versioning). So we first need to tell NewHELM that we have a dependency on that file by listing it in `get_dependencies`:
+### 1. Make the test items
+The first phase in a PromptResponseTest is making the `TestItem`s. We want these to be our questions from `an_example.jsonl`. NewHELM uses [DependencyHelper](../newhelm/dependency_helper.py) to ensure good hygiene of data dependencies (e.g. versioning). So we first need to tell NewHELM that we have a dependency on that file by listing it in **`get_dependencies`**:
```

Comment:
Here and throughout, why change from `code` to **`strong code`**?

Reply:
I was just bolding the Test method names that correspond to each step. I can undo that if you think the inconsistency is awkward.

Reply:
Ah, yeah, at least to me that wasn't clear. I prefer not bolding, but its not a strong preference.

Reply:
I'll undo it in that case